src/components/home/homeContainer.jsx

```diff
@@ -19,27 +19,42 @@ const Home = () => {
   };
 
   return (
-    <div>
-      <div className="row home-container w-100">
-        <div className="border col-3 principal-Container">
-          <h4>
-            {userName}
-          </h4>
-          <h4> NOMBRE DEL CONSULTORIO </h4>
-          <div className="border w-100">
-            {/* here must go the calendar */}
-          </div>
-          <button type="button" onClick={() => logOutFetch()}>
+    <div className="wrapper App">
+      <div className="row w-100 principal-Container">
+        <div className="border col-3" id="childenConteiner">
+          <h4 id="usercolor">{userName}</h4>
```

Comment:
why by id?? why not by className?

Reply:
at the beginning I had problems, the color of the component was deconfigured, that's why I had to do an id

Reply:
in line 25 put an id, because they are attributes that only that line will have